packages/starlight/components/Search.astro

```diff
@@ -294,6 +294,7 @@ const t = useTranslations(Astro.props.locale);
 
   #starlight__search .pagefind-ui__result-excerpt {
     font-size: var(--sl-text-body-sm);
+    word-break: break-all;
```

Comment:
Thanks for catching this Kevin!

I think we might prefer `break-word` here (always enjoyed the weird palindrome of this property–value pair 😅). Basically that lets word breaks _when necessary_ but not always:

```suggestion
    word-break: break-word;
```

| `break-all` | `break-word` |
|--------|--------|
| <img width="233" alt="image" src="https://github.com/withastro/starlight/assets/357379/7e1c34e8-08ac-4053-8acf-7850ee25589d"> | <img width="238" alt="image" src="https://github.com/withastro/starlight/assets/357379/115be8c8-c74e-40db-af6b-dc35f4a8906c"> |

Reply:
I see. I found that `word-break: break-word;` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values) and an alternative would be:

```css
word-break: normal;
overflow-wrap: anywhere;
```

It seems to work as well
<img width="332" alt="image" src="https://github.com/withastro/starlight/assets/46791833/dbd00a71-f721-4b45-9fc7-6c0992d7e384">

I am okay with both. I leave the final call to you ⭐

Reply:
```suggestion
    word-break: normal;
    overflow-wrap: anywhere;
```

Reply:
TIL!

I guess in this case we don't need to explicitly set `word-break: normal` as that's the default.

Also reading those CSS docs I think behaviour is the same in this context but it might be best to choose `break-word` instead of `anywhere` because of [how it impacts implicit content width calculation](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#values)
```suggestion
    overflow-wrap: break-word;
```